### PR TITLE
[testing] Cleanup.

### PR DESCRIPTION
**Pull request by mpimenov**

### CODE REVIEW

**Comment by mpimenov** [comment]
JTA

File: drape/drape_tests/testingmain.cpp [hunk 1]
```diff
@@ -43,32 +43,32 @@ int main(int argc, char * argv[])
...
   {
-    std::string fileName(pTest->m_FileName);
-    std::string testName(pTest->m_TestName);
+    std::string filename(test->m_filename);
+    std::string testname(test->m_testname);
     // Retrieve fine file name
     {
-      int iFirstSlash = static_cast<int>(fileName.size()) - 1;
-      while (iFirstSlash >= 0 && fileName[iFirstSlash] != '\\'  && fileName[iFirstSlash] != '/')
+      int iFirstSlash = static_cast<int>(filename.size()) - 1;
```
Comment (tatiana-yan) [inline drape/drape_tests/testingmain.cpp]: s/iFirstSlash/firstSlashIndex ?

File: drape/drape_tests/testingmain.cpp [hunk 2]
```diff
@@ -43,32 +43,32 @@ int main(int argc, char * argv[])
...
-        fileName.erase(0, iFirstSlash + 1);
+        filename.erase(0, iFirstSlash + 1);
     }
 
-    testNames.push_back(fileName + "::" + testName);
+    testnames.push_back(filename + "::" + testname);
     testResults.push_back(true);
   }
 
   int iTest = 0;
```
Comment (tatiana-yan) [inline drape/drape_tests/testingmain.cpp]: s/iTest/testIndex/g?

File: drape/drape_tests/testingmain.cpp (hunk 1 above)
Comment (mpimenov) [reply drape/drape_tests/testingmain.cpp]: done

File: drape/drape_tests/testingmain.cpp (hunk 2 above)
Comment (mpimenov) [reply drape/drape_tests/testingmain.cpp]: done

**Pull request merged**